### PR TITLE
log json object detail in routers

### PR DESCRIPTION
**Pull request by AlvaroVega**

### CODE REVIEW

File: CHANGES_NEXT_RELEASE [hunk 1]
```diff
@@ -1,3 +1,4 @@
+Fix router logs to print object details
```
Comment (fgalan) [inline CHANGES_NEXT_RELEASE:1]: No issue number associated?

Just checking...

**Comment by fgalan** [comment]
Lint error:

```
lib/models/notices.js: line 29, col 5, 'logger' is defined but never used.
```

**Comment by AlvaroVega** [comment]
fixed in https://github.com/telefonicaid/perseo-fe/pull/508/commits/65f73d5bf7ce98983b425d48fd1ddc12e0da0800

**Review by fgalan — APPROVED** [review]
LGTM

File: CHANGES_NEXT_RELEASE (hunk 1 above)
Comment (fgalan) [reply CHANGES_NEXT_RELEASE:1]: Confirmed by skype there isn't any associated issue. NTC.

**Pull request merged**